docs/install_guides/installing-openvino-docker-linux.md

```diff
@@ -1,68 +1,64 @@
+  * Linux
+  * Windows Subsystem for Linux 2 on CPU or GPU
+  * macOS on CPU only
+  
+  To launch a Linux image on Windows Subsystem for Linux 2, note the following things:
 
-## Build a Docker* Image
+  - Only Windows 10 with 21H2 update or above installed and Windows 11 are supported.
+  - Intel GPU driver on Windows host with version 30.0.100.9684 or above need be installed. Please see [this article](https://www.intel.com/content/www/us/en/artificial-intelligence/harness-the-power-of-intel-igpu-on-your-machine.html#articleparagraph_983312434) for more details.
+  - From 2022.1 release, the Docker images contain preinstalled recommended version of OpenCL Runtime with WSL2 support.
 
-You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci).
-The Framework can generate a Dockerfile, build, test, and deploy an image with the Intel® Distribution of OpenVINO™ toolkit.
-You can also try our [Tutorials](https://github.com/openvinotoolkit/docker_ci/tree/master/docs/tutorials) which demonstrate the usage of Docker containers with Intel® Distribution of OpenVINO™ toolkit. You can find device specific steps to configure an Intel® Distribution of OpenVINO™ toolkit Dockerfile below.
-
-## Use Docker* Image for CPU
-
-- Kernel reports the same information for all containers as for native application, for example, CPU, memory information.
-- All instructions that are available to host process available for process in container, including, for example, AVX2, AVX512. No restrictions.
-- Docker\* does not use virtualization or emulation. The process in Docker* is just a regular Linux process, but it is isolated from external world on kernel level. Performance penalty is small.
+@endsphinxdirective
 
-### <a name="configuring-for-cpu"></a>Configure a Docker* Image for CPU
+## Installation Flow
 
-You don't need to do specific steps to configure an Intel® Distribution of OpenVINO™ toolkit Dockerfile for CPU. You can use [available Dockerfiles](https://github.com/openvinotoolkit/docker_ci/tree/master/dockerfiles) or generate a Dockerfile with your setting via [DockerHub CI Framework](https://github.com/openvinotoolkit/docker_ci).
+There are two ways to install OpenVINO with Docker. You can choose either of them according to your needs:
+* Use a prebuilt image. Do the following steps:
+  1. <a href="#get-prebuilt-image">Get a prebuilt image from provided sources</a>.
+  2. <a href="#run-image">Run the image on different devices</a>.
+  3. <a href="#run-samples">(Optional) Run samples in the Docker image</a>.
+* If you want to customize your image, you can also build a Docker image manually by using the following steps:
+  1. <a href="#create-dockerfile">Create a Dockerfile</a>.
```

Comment:
Maybe replace it to the `Prepare a Dockerfile` or `Take a Dockerfile`? The Dockerfiles are already created, so users can modify it for their own purposes.

Reply:
"Prepare" sounds a better word for this. Updated. :)